tests/Doctrine/Tests/DBAL/Functional/Schema/PostgreSqlSchemaManagerTest.php

```diff
@@ -502,6 +502,34 @@ public function autoIncrementTypeMigrations() : array
             'bigint->int' => ['bigint', 'integer', 'INT'],
         ];
     }
+
+    /**
+     * @group 2925
+     */
+    public function testAlterTableChangePrimaryKey() : void
```

Comment:
Please move this test to `SchemaManagerFunctionalTestCase`. It should be checked and work the same for all platforms, not only for PostgreSQL.